lib/webauthn/authenticator_attestation_response.rb

```diff
@@ -98,5 +114,30 @@ def raw_subject_key_identifier(certificate)
       ext_value = ext_asn1.value.last
       OpenSSL::ASN1.decode(ext_value.value).value
     end
+
+    def attestation_root_certificates_store
+      certificates =
+        WebAuthn.configuration.attestation_root_certificates_finders.reduce([]) do |certs, finder|
+          if certs.empty?
+            finder.find(attestation_format: attestation_format,
+                        aaguid: aaguid,
+                        attestation_certificate_key_id: attestation_certificate_key) || []
+          end
```

Comment:
I think we need to return `certs` in the `else` clause here... :thinking: 

If we do and the tests didn't caught that, maybe we can consider adding a test case for that...?

Reply:
Added some tests!

Reply:
Thank you! :-)